src/constants.ts

```diff
@@ -5,6 +5,7 @@ declare global {
   interface Window {
     METAFLOW_SERVICE: string;
     FEATURES: FeatureFlags;
+    MF_NUM_DAYS: string;
```

Comment:
nit: I wonder if there is a more descriptive name, idk, `DEFAULT_TIME_FILTER_DAYS` or something